doc/doxygen/install/install-linux.doxygen

```diff
@@ -155,8 +155,11 @@
       \endcode
 
   To avoid problems while building the libraries contained in the contrib package,
-  the path to your contrib-build directory should not have any spaces in it.
-
+  the path to your contrib-build directory should not have any spaces in it. Just in case
+  you still have problems after building , try building ALL
```

Comment:
maybe change this to:

"In case your distribution does not provide some of the packages (or in different versions than expected), it is sometimes easier to build all dependencies manually which can solve issues with incompatible versions:"